<ui version="4.0" >
 <class>MarkersTab</class>
 <widget class="QWidget" name="MarkersTab" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="_markersGroup" >
     <property name="title" >
      <string>Axis Markers</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>New Marker Point</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLineEdit" name="_newMarker" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="_addMarker" >
              <property name="text" >
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Current Markers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="_currentMarkersList" >
            <property name="selectionMode" >
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="_clearMarkers" >
            <property name="text" >
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_removeMarker" >
            <property name="text" >
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_axisMajorGridGroup" >
     <property name="title" >
      <string>Marker Appearance</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="_Label_3" >
        <property name="text" >
         <string>Line Style</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="_markerLineStyle" />
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Line Color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="Kst::ColorButton" name="_markerLineColor" />
      </item>
      <item row="0" column="4" >
       <widget class="QLabel" name="label_11" >
        <property name="text" >
         <string>Line Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" >
       <widget class="QDoubleSpinBox" name="_markerLineWidth" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Automatic Markers</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="_curveMarkers" >
        <property name="text" >
         <string>Use Curve as marker source:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2" >
       <widget class="Kst::CurveSelector" native="1" name="_curve" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Create makers on:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QRadioButton" name="_risingEdge" >
        <property name="text" >
         <string>Rising Edge</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QRadioButton" name="_fallingEdge" >
        <property name="text" >
         <string>Falling Edge</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QRadioButton" name="_both" >
        <property name="text" >
         <string>Both</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QCheckBox" name="_vectorMarkers" >
        <property name="text" >
         <string>Use Vector as marker source:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2" >
       <widget class="Kst::VectorSelector" native="1" name="_vector" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Kst::ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::VectorSelector</class>
   <extends>QWidget</extends>
   <header>vectorselector.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::CurveSelector</class>
   <extends>QWidget</extends>
   <header>curveselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
